utils/common

```diff
@@ -91,4 +93,46 @@ extract_value_from_json() {
     fi 
 }
 
-create_logs_dir
+extract_input_params_from_file() {
+
+  # Check if the input file has a JSON extension
+  if [[ "$2" != *.json ]]; then
+      status_log $TYPE_ERROR "Error: Input file is not a JSON file, please run \'grpl c i h\'  to get more info on how to pass agruments from cli"
+      exit 1
+  fi
+
+
+  json=$(<${2})
+
+  valid_args=$1
+  for val in "${valid_args[@]}"; do
+      export "$val=$(extract_value_from_json "$json"  "$val")"
+  done
+
+}
+
+
+extract_input_params_from_cli() {
+
+  valid_args=$1
+  shift
+  # Loop through arguments
+  while [[ $# -gt 0 ]]; do
+
+      arg1=$1
+      shift
+
+      config_param="${arg1#--}"
+      IFS='=' read -r key value <<< $config_param
+      # Check if the argument is valid
+      if [[ " ${valid_args[@]} " =~ " $key " ]]; then
+          export $config_param
+      else
+          status_log $TYPE_ERROR "$key is not a valid argument, please run \'grpl c i h\'  to get more info on how to pass agruments from cli"
```

Comment:
@zahid-iqbal-alpha 
in the text: "please run \'grpl c i h\'"
this is a central place, I think "grpl c i h" is too specific for the example.